docs/UserGuide.md

```diff
@@ -62,116 +62,68 @@ Jobby comes equipped with a [GUI]() interface that provides visual feedback to y
 Hello and welcoem to Jobby! We are delighted that you've chosen our platform to manage your internship application process!
 
 Before we begin, please ensure that you have viewed the following sections of the guide:
-* [Quick Start]() to help you get Jobby up and running
-* [Navigating the Interface]() to get you familiarised with navigating Jobby
+* [Installation](#installation) to help you get Jobby up and running.
+* [Navigating the Interface](#navigating-the-interface) to get you familiarised with navigating Jobby.
 
 This tutorial aims to guide you through the most essential features and functionalities of Jobby that you will use on a day-to-day
 basis. In this guide, you will learn how to add your first tutor and corresponding schedule!
 
 Click on any of the hyperlinks below to jump to respective sections of the tutorial!
-* Beginning
-* Adding your first contact
-* Adding your first application
+* [The Beginning](#beginning)
+* [Adding your first Organization](#adding-your-first-organization)
+* [Adding your first Recruiter](#adding-your-first-recruiter)
+* [Adding your first Application](#adding-your-first-application)
 
-Jobby comes equipped with a [GUI]() interface that provides visual feedback to you. Below is a quick overview of the various components of our interface.
-
-![ui overview](images/ug-images/labelled-gui.png)
-
-### Adding `Contact`
-Jobby can create `Contacts`. In the form of `Organization` and `Recruiter`, which you can add into your database with simple CLI commands.
-
-Let's say you want to track your job application for the `Organization: Woogle`, with a `Recruiter: John`.
-
-You can add the `Organization` into Jobby with a simple command:
-```add --org --name Woogle --id woogle_id```
-
-![Add Organization](images/starter-guide/add-woogle.jpg)
-
-
-You can then add a **recruiter** that's related to the "Woogle" organization like so::
-```add --rec --name John --id john_id --oid woogle_id```
-
-![Add Recruiter](images/starter-guide/add-john.jpg)
-
-
-> To find out all of `Organization` and `Recruiter`'s flags, go [organization documentation]()
-> and [recruiter documentation]()
-
-### Adding `Application`
-Let's say now you want to apply to `Woogle` for the position of `Junior Software Engineer` role. You will be able to create
-an `Application` for this company with the by including the `Organization ID` in the command:
-`apply woogle_id --title Software Engineer --by 22-11-2023`
-
-![Add Application](images/starter-guide/add-application.jpg)
+### The Beginning
+When you first launch Jobby, you will notice that it comes preloaded with sample data for you to play with. You may use 
+the sample data to familiarise yourself with the various [features](#features) that Jobby provides.
```

Comment:
Linking to the features immediately may be countering the purpose of the starter guide. We should discuss if this is really necessary or if there's a better way to phrase this.

Reply:
I resolved it.